src/fragments/lib-v1/utilities/js/hub.mdx

```diff
@@ -0,0 +1,94 @@
+Amplify has a local eventing system called Hub. It is a lightweight implementation of Publisher-Subscriber pattern, and is used to share data between modules and components in your app. Amplify uses Hub for different categories to communicate with one another when specific events occur, such as authentication events like a user sign-in or notification of a file download.
```

Comment:
This file is a direct copy from `lib` with minor edits (almost entirely removals) to make it V6 compatible.